### PR TITLE
css for TOC

### PR DESCRIPTION
**Pull request by Ivorson**
Sorry I forgot the CSS counterpart for my last TOC pull request, here is the stylesheets and the related modification on js/html

### CODE REVIEW

File: manifest.json [hunk 1]
```diff
@@ -61,7 +65,10 @@
       "js/config.js",
       "js/runMathJax.js",
       "js/marked.js",
-      "js/options.js"
+      "js/options.js",
+	  "js/jquery.js",
+	  "js/markdownTOC.css",
```
Comment (volca) [inline manifest.json]: Is this line a typo?

**Comment by volca** [comment]
Can we merge GitHub.css and Github-left.css? There're so many theme css files yet.

File: manifest.json (hunk 1 above)
Comment (Ivorson) [reply manifest.json]: yeah...

**Comment by Ivorson** [comment]
Yes. I copy GitHub.css as Github-left.css, just to let GitHub.css untouched

**Pull request merged**